remotedev/create.py

```diff
@@ -201,10 +202,15 @@ def print_completion(username):
     # set user_data
     user_data = set_user_data(username=args.username, userkeys=public_keys)
 
+    tags = args.tags
+    if not tags:
+        tags = [args.username]
```

Comment:
I'm not sure whether we really need the username as a tag as a default at all, since the name will have the username in it anyway @rishig thoughts?